lib/src/fade.dart

```diff
@@ -84,17 +92,18 @@ class FadeAnimatedTextKit extends AnimatedTextKit {
         );
 
   static List<AnimatedText> _animatedTexts(
-    List<String> text,
-    TextAlign textAlign,
-    TextStyle textStyle,
-    Duration duration,
-  ) =>
+          List<String> text,
+          TextAlign textAlign,
+          TextStyle textStyle,
+          Duration duration,
+          double fadeInEnd,
+          double fadeOutStart) =>
```

Comment:
Again, the trailing comma should be used so that indentation doesn't impact lines that weren't changed.

Reply:
Done